packages/faustwp-core/src/utils/adminUrl.ts

```diff
@@ -0,0 +1,13 @@
+import { untrailingslashit } from './untrailingslashit.js';
+
+/**
+ * Retrieves the URL to the admin area for the current site.
+ *
+ * @link https://developer.wordpress.org/reference/functions/admin_url/
+ */
+export function adminUrl(path = ''): string {
+  const baseUrl: string = process.env.NEXT_PUBLIC_WORDPRESS_URL || '';
```

Comment:
We should also get the `baseUrl` from `getWpUrl()` to respect the filters we have in place.

Reply:
Absolutely! Good catch!